### PR TITLE
Add Atharva Shirdhankar to the list

### PR DESCRIPTION
**Pull request by StarTrooper08**

### CODE REVIEW

File: public/data/StarTrooper08.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/StarTrooper08",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/_Atharva_08",
+      "icon": "twitter"
+    },
```
Comment (Panquesito7) [inline public/data/StarTrooper08.json:15]:
```suggestion
    }
```

File: src/Components/Links.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ function Links({ links }) {
     github: '#171515',
     instagram: '#E4405F',
     linkedin: '#0077b5',
+
```
Comment (Panquesito7) [inline src/Components/Links.js:15]: Please revert the changes in this file.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: src/Components/Links.js (hunk 2 above)
Comment (StarTrooper08) [reply src/Components/Links.js:15]: ok

**Comment by StarTrooper08** [comment]
Did the suggested changes

**Review by eddiejaoude — APPROVED** [review]
Thanks

**Pull request merged**